src/index.html

```diff
@@ -4,15 +4,89 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     />
     <title>Stars</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
```

Comment:
The link element for the stylesheet should be formatted with each attribute starting on a new line, as there are more than two attributes. This will improve readability.